device/vxlan/unicast.yaml

```diff
@@ -0,0 +1,45 @@
+components:
+  schemas:
+    Vxlan.Unicast:
+      description: >-
+        Configuration of Unicast parameters for VxLAN.
```

Comment:
Whole unicast is not required,if required the description should change to "Static REmote VM Arp entries"

Reply:
As discussed, the communication of VM information as can be done using either multicast or unicast. keeping the destionation mode as is.